lib/view_model/settings/settings_view_model.dart

```diff
@@ -264,55 +69,31 @@ abstract class SettingsViewModelBase with Store {
 
   @computed
   TransactionPriority get transactionPriority {
-    final priority = _settingsStore.priority[_walletType];
+    final priority = _settingsStore.priority[walletType];
 
     if (priority == null) {
-      throw Exception('Unexpected type ${_walletType.toString()}');
+      throw Exception('Unexpected type ${walletType.toString()}');
     }
 
     return priority;
   }
```

Comment:
only the Exchange view model uses this getter and it has a dependency for `settingsStore` too, so we can move it to the ExchangeViewModel and remove its dependency on this class